resources/chat-view/chat.js

```
@@ -0,0 +1,499 @@
+  function toggleTreeItemBoxes() {
+    const pipelineRunsBox = document.querySelector(
+      'input[type="checkbox"][value="pipelineContext"]'
+    );
+
+    const pipelineRunBoxes = document.querySelectorAll(
+      'input[type="checkbox"][value*="Pipeline Run:"]'
+    );
+
+    // Condition: If the main checkbox is clicked, all children checkboxes should be checked.
+    if (pipelineRunsBox.checked) {
+      pipelineRunBoxes.forEach(checkbox => (checkbox.checked = true));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)